test/server/cards/10-TEL/ChampionsOfYomi.spec.js

```diff
@@ -0,0 +1,168 @@
+fdescribe('Champions of Yomi', function() {
```

Comment:
Need to remove `fdescribe`

Reply:
Right, I was in a hurry when committing the requested change this morning, sorry about that.